homeassistant/components/ezviz/translations/ar.json

```diff
@@ -1,7 +1,7 @@
 {
     "config": {
         "abort": {
-            "ezviz_cloud_account_missing": "\u062d\u0633\u0627\u0628 \u0633\u062d\u0627\u0628\u0629 Ezviz \u0645\u0641\u0642\u0648\u062f. \u064a\u0631\u062c\u0649 \u0625\u0639\u0627\u062f\u0629 \u062a\u0643\u0648\u064a\u0646 \u062d\u0633\u0627\u0628 \u0633\u062d\u0627\u0628\u0629 Ezviz"
+            "ezviz_cloud_account_missing": "\u062d\u0633\u0627\u0628 \u0633\u062d\u0627\u0628\u0629 EZVIZ \u0645\u0641\u0642\u0648\u062f. \u064a\u0631\u062c\u0649 \u0625\u0639\u0627\u062f\u0629 \u062a\u0643\u0648\u064a\u0646 \u062d\u0633\u0627\u0628 \u0633\u062d\u0627\u0628\u0629 EZVIZ"
```

Comment:
Translations are managed via lokalize.
You should only update the English translation.

Reply:
Addressed in 0d858f189294e18df929d4f3eb1061bfcbd475a8.